ansible/logs.yml

```diff
@@ -24,14 +24,13 @@
         - subjects
       when: "'db' not in exclude_logs_from"
 
-- hosts: all:!ansible
-  serial: 1
+- hosts: shared,core,invokers
   tasks:
   - name: get all docker containers
-    local_action: shell docker --host tcp://{{ ansible_host }}:{{ docker.port }} ps -a --format="{% raw %}{{.Names}}{% endraw %}"
```

Comment:
isn't this semantic changing?

Reply:
Good catch, I will leave it working as is now for linux and docker-machine and use `--host tcp://{{ ansible_host }}:{{ docker.port }}` and for Docker for Mac it will not present.